app/src/main/res/layout/full_screen_image_dialog.xml

```diff
@@ -0,0 +1,30 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+             xmlns:app="http://schemas.android.com/apk/res-auto"
+             android:layout_width="match_parent"
+             android:layout_height="match_parent"
+             android:background="@android:color/black"
+             android:orientation="vertical">
+
+    <com.davemorrissey.labs.subscaleview.SubsamplingScaleImageView
+        android:id="@+id/subsampling_image_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        />
+
+    <me.zhanghai.android.materialprogressbar.MaterialProgressBar
+        android:id="@+id/mpb_loading_progress"
+        android:layout_width="60dp"
+        android:layout_height="60dp"
+        android:layout_gravity="center"
+        app:mpb_progressStyle="circular"
+        />
```

Comment:
We use these dimensions (60dp) of progress bar in 3 places, this should go to dimens.xml (or styles.xml if you want to move all progress bar properties to one place)